.github/workflows/release-image.yaml

```diff
@@ -0,0 +1,46 @@
+---
+name: Release server container image
```

Comment:
This is pretty much the same as `.github/workflows/build-image.yaml`. Is there a way to reuse the existing build image action spec adding the `on.workflow_dispatch` to `.github/workflows/build-image.yaml`?

Reply:
I don't know how to compose these workflows… do you have any pointers? I'll look around

Reply:
Ok, I'll try a `composite` action, refactoring the common bits in `.github/actions/quay.io-publish.yaml` or something. This might need a few trials tho 🤦